models/order/src/builder.spec.ts

```diff
@@ -303,20 +186,22 @@ describe('builder', () => {
           typeId: 'quote',
           __typename: 'Reference',
         }),
-        custom: null,
-        paymentInfo: null,
-        locale: expect.any(String),
-        inventoryMode: expect.any(String),
-        shippingRateInput: null,
-        origin: null,
-        itemShippingAddresses: expect.arrayContaining([
-          expect.objectContaining({
-            title: expect.any(String),
-            state: expect.any(String),
-          }),
-        ]),
+        stateRef: expect.objectContaining({
+          typeId: 'state',
+          __typename: 'Reference',
+        }),
+        createdBy: expect.objectContaining({
+          customerRef: expect.objectContaining({ typeId: 'customer' }),
+          userRef: expect.objectContaining({ typeId: 'user' }),
+        }),
+        lastModifiedBy: expect.objectContaining({
+          customerRef: expect.objectContaining({ typeId: 'customer' }),
+          userRef: expect.objectContaining({ typeId: 'user' }),
+        }),
         __typename: 'Order',
       })
     )
   );
 });
+
+//TODO: add tests when customizing
```

Comment:
Instead of adding more tests, perhaps it's easier if we just extend the ones we already have to provide not only the Order with the default values but also configure the optional fields.

Reply:
Tests have been updated here: 145fd9bd1b120fefc79d575e80184bae01eb9209